.travis.yml

```diff
@@ -128,59 +128,14 @@ jobs:
         - TID=Manual Install (GRDBCipher, Xcode 9.4)
       script: make test_install_GRDBCipher
 
-    # CocoaPods Lint (Xcode 9.4)
+    # Cocoapods Lint (Xcode 9.4)
     - stage: Test Installation
       osx_image: xcode9.4
       install:
         - gem install cocoapods # Since Travis is not always on latest version
       env:
-        - TID=CocoaPods Lint GRDB.swift (Xcode 9.4)
```

Comment:
Oops, it looks like you are no longer developing against the development branch. You are about to revert #393 :-)